apollos-church-api/package.json

```diff
@@ -42,9 +42,9 @@
     "randomcolor": "0.5.3"
   },
   "devDependencies": {
-    "@apollosproject/apollo-server-env-mock": "^2.27.2",
-    "@apollosproject/babel-preset-apollos": "^2.27.2",
-    "@apollosproject/eslint-config": "^2.27.2",
+    "@apollosproject/apollo-server-env-mock": "^2.29.3",
```

Comment:
why are these on different versions

Reply:
Release 2.29.3 changed them

Reply:
No i mean why didn't they get bumped to 2.30

Reply:
Latest version of the template also has them at 2.29.3. Not sure why.

Reply:
Oh weird